client/src/components/RichTextEditor.js

```diff
@@ -130,6 +150,16 @@ const exporterConfig = {
     }
 
     return originalText
+  },
+
+  styleToHTML: style => {
+    if (style === INLINE_STYLE.STRIKETHROUGH) {
+      return <strike />
+    }
```

Comment:
So by your comment https://github.com/NCI-Agency/anet/pull/2790/files#r375806248 , it this then still needed?

Reply:
Indeed, this check is obsolete and can be safely removed. It comes from a previous version of the editor which required dedicated HMTL style parsing.

Reply:
Just to be clear: `<mark>` is supported out of the box (both ways). `<strike>` needs parsing when converting the editor style to HMTL.